server/routes/appointments/attendance/handlers/summaries.test.ts

```diff
@@ -6,13 +6,16 @@ import ActivitiesService from '../../../../services/activitiesService'
 import { AppointmentAttendanceSummary } from '../../../../@types/activitiesAPI/types'
 import DateOption from '../../../../enum/dateOption'
 import { parseIsoDate } from '../../../../utils/datePickerUtils'
+import PrisonService from '../../../../services/prisonService'
+import { Prisoner } from '../../../../@types/prisonerOffenderSearchImport/types'
 
 jest.mock('../../../../services/activitiesService')
 
 const activitiesService = new ActivitiesService(null) as jest.Mocked<ActivitiesService>
+const prisonService = new PrisonService(null, null, null) as jest.Mocked<PrisonService>
 
 describe('Route Handlers - Appointment Attendance Summaries', () => {
-  const handler = new SummariesRoutes(activitiesService)
+  const handler = new SummariesRoutes(activitiesService, prisonService)
```

Comment:
I can't see any test coverage where the prisoner-service (mock) is used to assert on the prisoner details?